docs/api.md

```diff
@@ -66,9 +66,19 @@ Wrap the `HomeRoute` in an `AuthenticatedRoute` to specify the route you want to
 Route that when used, requires that a session is established before continuing. Else redirects the user to the `LoginRoute` path.
 
 ```html
-<AuthenticatedRoute path='/home/protected' component={RegisterPage} />
+<AuthenticatedRoute path='/profile' component={ProfilePage} />
 ```
 
+Specify the option `inGroup` to only allow users in a certain group to access the path.
+
+```html
+<AuthenticatedRoute path='/admin' inGroup="administrator" component={AdminPage} />
+```
+
+If you want to specify more complex groups, please see section [Group Expressions](#group-expressions).
+
+**Important:** In order to use the `inGroup` option, you must expand the `groups` resource for the `/me` endpoint.
```

Comment:
I would probably link to the documentation on how to do that (as it's not the most obvious):

* <https://docs.stormpath.com/nodejs/express/latest/user_data.html#current-user-json-api>
* <https://docs.stormpath.com/nodejs/express/latest/user_data.html#automatic-expansion>

*(Note: the second link needs to be updated because it's no longer correct.)*